src/lcms-client.js

```diff
@@ -5,6 +5,7 @@ async function getLearningContent(configuration) {
   const url = configuration.LCMS_API_URL + '/databases/airtable';
   const requestConfig = {
     headers: { 'Authorization': `Bearer ${configuration.LCMS_API_KEY}` },
+    timeout: 10000,
```

Comment:
```suggestion
    timeout: 60000,
```

Le timeout de 10 secondes me parait un peu cours